Issue 66

Mifrill opened the pull request:
fixes #66

Mifrill committed:
update bundle gems

Mifrill committed:
rails 5.0.2

Mifrill committed:
update rails to 5.1

Mifrill committed:
rspec done

Mifrill commented:
Using actionpack 5.1.0.rc2 (was 5.0.2)
Using actioncable 5.1.0.rc2 (was 5.0.2)
Using actionmailer 5.1.0.rc2 (was 5.0.2)
Using railties 5.1.0.rc2 (was 5.0.2)
Using arel 8.0.0 (was 7.1.4)
Using activesupport 5.1.0.rc2 (was 5.0.2)
Using activemodel 5.1.0.rc2 (was 5.0.2)
Using activejob 5.1.0.rc2 (was 5.0.2)
Using activerecord 5.1.0.rc2 (was 5.0.2)
Using actionview 5.1.0.rc2 (was 5.0.2)

Mifrill committed:
versions

zharikovpro committed:
gem database_cleaner updated to remove some deprecation warnings related to Rails 5.1 upgrade

zharikovpro committed:
do not comment db extensions to avoid heroku CI failures as script do not own extensions

zharikovpro committed:
removed old hack to run Rails on the port 5000 as default, no need for that

Mifrill committed:
Merge branch 'issue_66' of github.com:zharikovpro/railgun into issue_66

zharikovpro committed:
ActiveAdmin 1.0

zharikovpro committed:
install inherited_resources from rubygems, not repo

Mifrill committed:
nokogiri

Mifrill committed:
nokogiri

Mifrill committed:
nokogiri

Mifrill committed:
Merge branch 'issue_66' of github.com:zharikovpro/railgun into issue_66

zharikovpro reviewed (changes requested):
need rails 5.1.x (not rc2)

zharikovpro committed:
upd devise and lograge - compatible with Rails 5.1

Mifrill committed:
update some gems

Mifrill committed:
schema_validations

Mifrill committed:
merge

Mifrill committed:
repiar

zharikovpro commented:
You are trying to install in deployment mode after changing
       your Gemfile. Run `bundle install` elsewhere and add the
       updated Gemfile.lock to version control.
       
       You have added to the Gemfile:
       * rails (~> 5.1.0)
       
       You have deleted from the Gemfile:
       * rails (~> 5.1.0.rc2)
 !
 !     Failed to install gems via Bundler.

Mifrill committed:
repiear right version rails

zharikovpro committed:
removed schema_validations

reasons:

1) not actively maintained, rails 5.1 PR hanged for 18 days
2) increases coupling with db layer, doesn't suppor mongo, etc
3) forces to use ActiveModel, can't use ROM, Validation objects. etc
